pkg/postgres/query/query.go

```diff
@@ -35,19 +35,28 @@ func getQuotedName(fieldName string) (string, error) {
 
 func simpleOperatorCondition(
 	field filter.RequestField, valueIsList bool, singleValueTemplate string, listValueTemplate string,
-) (conditionTemplate string, conditionParams []any, err error) {
+) (conditionTemplate string, err error) {
 	quotedName, err := getQuotedName(field.Name)
 	if err != nil {
-		return "", nil, errors.Wrap(err, "could not get quoted name")
+		err = errors.New("could not get quoted name")
```

Comment:
the original errors is currently dropped, but should be wrapped via `fmt.Errorf` instead

Reply:
This change has been reverted to:
`return "", nil, errors.Wrap(err, "could not get quoted name")`